runtime/fundamentals/node.md

```diff
@@ -16,12 +16,20 @@ oldUrl:
 - /runtime/manual/node/private_registries
 ---
 
-Modern Node.js projects will run in Deno with little to no reworking required.
-However, there are some key differences between the two runtimes that you can
-take advantage of to make your code simpler and smaller when migrating your
-Node.js projects to Deno.
+- **Deno is fully Node-compatible**. Virtually any modern Node project will run
+  in Deno with little or no change!
```

Comment:
Having "fully" and then "virtually" together seems like a slight contradiction.
I think we can simplify this a touch and sound less like we are equivocating. How about this, which removes the "virtually" caveat and refocusses on how most project require little or no change?

```suggestion
- **Deno is Node-compatible**. Most Node projects will run in Deno with little
  or no change!
```